core/domain/exp_services_test.py

```diff
@@ -1305,6 +1305,49 @@ def test_unpublished_explorations_can_accept_commit_message(self):
             ), None)
 
 
+class UpdateExplorationContributionDatetimeTests(ExplorationServicesUnitTests):
```

Comment:
once moved to user_service_test, should the code be included in class UserServicesUnitTests ?